modules/nodes-sigstore-configure-cluster-policy.adoc

```diff
@@ -6,13 +6,13 @@
 [id="nodes-sigstore-configure-cluster-policy_{context}"]
 = Creating a cluster image policy CR
 
-A `ClusterImagePolicy` custom resource (CR) enables a cluster administrator to configure a Sigstore signature verification policy for the entire cluster. When enabled, the Machine Config Operator (MCO) watches the `ClusterImagePolicy` object and updates the `/etc/containers/policy.json` file in the cluster.
+A `ClusterImagePolicy` custom resource (CR) enables a cluster administrator to configure a Sigstore signature verification policy for the entire cluster. When enabled, the Machine Config Operator (MCO) watches the `ClusterImagePolicy` object and updates the `/etc/containers/policy.json` and `/etc/containers/registries.d/sigstore-registries.yaml` files on all the nodes in the cluster.
 
-The following example shows general guidelines on how to configure a `ClusterImagePolicy` object. For more details on the parameters, see "Configuring Sigstore support".
+The following example shows general guidelines on how to configure a `ClusterImagePolicy` object. For more details on the parameters, see "About cluster and image policy parameters".
```

Comment:
I think the it's more accurate to call "About cluster image policy and image policy parameters"